tests/stubtest_allowlists/py3_common.txt

```diff
@@ -424,8 +424,8 @@ typing\.AsyncGenerator
 typing\.AbstractSet
 
 # Internal attributes
-.*\.__protocol_attrs__
-.*\.__callable_proto_members_only__
+typing_extensions\..*\.__protocol_attrs__
+typing_extensions\..*\.__callable_proto_members_only__
```

Comment:
Not sure there's much point worrying about these too much, since the next mypy release will include https://github.com/python/mypy/commit/b8f8d800fc1d72452af67aa0ae64c48bfacb0c92

Reply:
Yeah, I realised this was because I had latest mypy installed